docs/samples/browser-plugin-meetings/app.js

```diff
@@ -651,9 +651,11 @@ async function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevi
       viewBreakouts();
     });
 
-        meeting.on('meeting:stoppedSharingRemote', () => {
-          meetingStreamsRemoteShare.srcObject = null;
-        });
+    meeting.on('meeting:stoppedSharingRemote', () => {
+      const remoteShareTemp = meetingStreamsRemoteShare.srcObject;
+      meetingStreamsRemoteShare.srcObject = null;
+      meetingStreamsRemoteShare.srcObject = remoteShareTemp;
```

Comment:
Could you please explain how this fixes the issue?

Reply:
Earlier we were simply assigning the null to the video element which was removing the remote media reference added during `addMedia()` call. Here we first remove the media to stop showing the remote stream being received and again attach the remote media stream but since the remote is not sending the frames anymore it does not show the screenshare but keeps the event attached. Hence, when we again start sharing the screen, the media flows correctly to the video element.

Reply:
it would be good to add this explanation as a comment in the code

Reply:
Yes. Please add a short comment explaining it and also does this work in both "Stop Share" and "Unpublish" cases?

Reply:
> does this work in both "Stop Share" and "Unpublish" cases?

Yes

Reply:
Added comment.